templates/blog/macros.html

```diff
@@ -10,7 +10,6 @@
       <div>
         <a href={{ url_for_user(article.author) }}>{{ article.author.username }}</a>
         <time datetime={{ article.creation_date }} pubdate>{{ article.creation_date.strftime('%d %B, %Y') }}</time>
-        <p class=reading-time><span>{{ (article.content|length / 6 / 250)|int }}</span> min read</p>
```

Comment:
could you try to do `article.content|striptags|length / 6 / 250)|int` and see what the reading time is with an image?